extractor/docker/docker_test.go

```diff
@@ -95,6 +95,27 @@ func TestApplyLayers(t *testing.T) {
 					},
 					WhiteoutFiles: []string{"app/composer.lock"},
 				},
+				{
+					SchemaVersion: 1,
+					ID:            "sha256:a3ed95caeb02ffe68cdd9fd84406680ae93d633cb16422d00e8a7c22955b46d4",
```

Comment:
can we change the layer SHAs to be something like: `sha256:openssl_and_musl_updated` in this case. It's much easier to debug.

Reply:
I understand what you mean, but I prefer a similar value to an original value. I think there are two advantages to it. The first one is it shows other developers what value is expected. We can easily understand how it works by checking test cases. As you can see, most libraries use the actual value.
https://github.com/google/go-containerregistry/blob/2be26e624e32c4b3de62287e153e5e382a79b7c5/pkg/crane/filemap_test.go#L30
https://github.com/containers/image/blob/21244c96ad792ef415068dc1bc1ab82dffb68dc3/storage/storage_image_test.go#L14

Another one is we can test the value properly. When I used a descriptive value before, I introduced some bugs since the function has the validation that the descriptive value passed, while the actual value failed. After that, I prefer the actual value.